server/src/main/resources/keywhiz-development.yaml

```diff
@@ -82,7 +82,7 @@ userAuth:
 # alternateUiPath: ui/app/
 
 # Base64 of "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
```

Comment:
This comment probably needs to be updated. Do you want to recommend a way to generate the the file? `head -c 32 /dev/urandom | base64`?